pkg/asset/lbconfig/lbconfig.go

```diff
@@ -0,0 +1,156 @@
+package lbconfig
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"path/filepath"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/installer/pkg/asset"
+)
+
+// LoadBalancerCategory describes the type of load balancer data.
+type LoadBalancerCategory string
+
+var (
+	// ConfigName is the name of the load balancer config file.
+	ConfigName = "openshift-lbconfig-custom-dns.yaml"
+
+	// ConfigPath is the path to the load balancer config file.
+	ConfigPath = filepath.Join("openshift", ConfigName)
+
+	// PublicLoadBalancer indicates that the data is related to public load balancers.
+	PublicLoadBalancer LoadBalancerCategory = "external-api-lb-dns-name"
```

Comment:
"external-api-lb-dns-name" and "internal-api-lb-dns-name" are artifacts of the AWS implementation. Can we change this to be "external-api-lb-ips" & "internal-api-lb-ips" or "external-api-lb-config" & "internal-api-lb-config"?